content/pages/changelog/index.md

```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-07-16 / Change to enterprise events for content access
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new content access events.
+
+### New event
+
+* A new `CONTENT_ACCESS` event is triggered when a file was accessed by an
```

Comment:
@jcleblanc do we have an example of the event body? Might it be worth adding this here?